pages/hospitals.php

```diff
@@ -0,0 +1,88 @@
+<?php include("../templates/header.php") ?>
```

Comment:
can u rename this file to `displayRegHosp.php`?

Reply:
Ook ,I will.